app/assets/stylesheets/common.scss

```diff
@@ -516,6 +511,23 @@ body.small-nav {
   }
 }
 
+@mixin dark-map-color-scheme {
+  .leaflet-tile-container .leaflet-tile,
+  .mapkey-table-entry td:first-child > * {
+    filter: brightness(.8);
```

Comment:
Doesn't the mentioned step 5 include the need for the filters to be set individually to give the cartographers a choice?
I don't see that as a necessity for this PR, but already setting the filter through a custom property defined on the layer could ease the transition to that being implemented and also act as default and fallback:
```css
here {
    filter: var(--dark-map-filter);
}
somewhere else {
    .leaflet-layer {
        --dark-map-filter: brightness(.8);
    }
}
```

Reply:
Step 5 has an impossible requirement of "make it dark but don't pick any filter". The closest possible thing is "don't touch existing filters".